freqtrade/data/dataprovider.py

```diff
@@ -36,13 +38,19 @@ def refresh(self,
             self._exchange.refresh_latest_ohlcv(pairlist)
 
     @property
-    def available_pairs(self) -> List[Tuple[str, str]]:
+    def available_pairs(self) -> ListPairsWithTimeframes:
         """
         Return a list of tuples containing (pair, timeframe) for which data is currently cached.
         Should be whitelist + open trades.
         """
         return list(self._exchange._klines.keys())
 
+    def create_pair_list(self, pairs: List[str], timeframe: str = None) -> ListPairsWithTimeframes:
+        """
+        Create list of pair tuples with (pair, ticker_interval)
+        """
+        return [(pair, timeframe or self._config['ticker_interval']) for pair in pairs]
+
```

Comment:
I would've moved this to pairlistmanager - it seems better suited there, as this does not provide any data ... (and a user will not want to use this, but will want to use `current_whitelist()`.

Having this available might be confusing to a user, who won't know if this or the other function is the correct one to ge the current pairlist ...

In the end, this is just a small helper function which combines pairs with timeframe ... but you need to have the list of pairs beforehand ...

Reply:
nice place for this function (and the datatype fits well into this module as well) 👍